packages/toolboxes/substrate/src/toolbox/baseSubstrateToolbox.ts

```diff
@@ -15,7 +15,11 @@ import { Network, type SubstrateNetwork } from "../types/network.ts";
 type SubstrateTransferParams = {
   recipient: string;
   assetValue: AssetValue;
-  from?: string;
+  from: string;
```

Comment:
IMO - it shouldn't be forced to pass `from` as transfer params as it supposed to be already "calculated" on connection 🤔

Reply:
done